src/emulator/downloadableEmulators.ts

```diff
@@ -57,9 +57,9 @@ const EMULATOR_UPDATE_DETAILS: { [s in DownloadableEmulators]: EmulatorUpdateDet
   dataconnect:
     process.platform === "darwin"
       ? {
-          version: "1.3.5",
```

Comment:
This wont be relevant for long, but we should remove this for now. Will dart be included in the next normal emulator release, or is it hidden behind an experiment

Reply:
Removed these changes. Thanks for the find!